CameraXBasic/app/src/main/res/layout/camera_ui_container.xml

```diff
@@ -17,10 +17,12 @@
 <androidx.constraintlayout.widget.ConstraintLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/camera_ui_container"
     android:layoutDirection="ltr"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    tools:background="@android:color/black">
```

Comment:
Please remove this, I don't think hard-coding the background color is a good idea here.

Reply:
This is for the preview only ...those buttons have a light color - and with a black background, there is contrast.